tests/runtimes/simple-keyvalue/src/types.rs

```diff
@@ -40,6 +40,20 @@ pub struct KeyValue {
     pub value: String,
 }
 
+#[derive(Clone, Debug, Default, cbor::Encode, cbor::Decode)]
+pub struct Encrypt {
+    pub epoch: u64,
+    pub key_pair_id: String,
+    pub plaintext: String,
```

Comment:
Would probably be easier to make these `Vec<u8>` (and `[]byte` on the Go side) instead of using base64.